src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj

```diff
@@ -1,15 +1,68 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.Azure.Management.Monitor</PackageId>
     <Description>Microsoft Azure Monitor Library</Description>
     <VersionPrefix>0.16.0-preview</VersionPrefix>
```

Comment:
This is a rename of the previous SDK dll, Microsoft.Azure.Insights.dll, with one breaking change for Metrics/MetricDefinitions API. The last version for Insights was 0.15.X.X-preview, so I changed the version to 16 to emphasize that the two major changes: name and Metrics API. All the other APIs in the dll remain unchanged.
This is the first time I will publish our SDK with this name, so I kept the version at 16.